apps/barista-design-system/src/components/ba-decision-graph/ba-decision-graph-start-node/ba-decision-graph-start-node.scss

```
@@ -0,0 +1,70 @@
+@import '../../../styles/global/variables';
+@import 'colors';
+
+:host {
+  display: flex;
+  margin-bottom: 20px;
+}
+
+:host ::ng-deep button {
+  font-weight: normal;
+}
+
+:host ::ng-deep .ba-uxd-startnode p {
+  margin-bottom: 20px;
+}
+
+.ba-uxd-startnode-grid {
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
```

You don't need to use the repeat function, if you are not repeating anything. This is a shorthand so you would not have to write `1fr, 1fr, 1fr, 1fr, 1fr` and instead can write `repeat(5, 1fr)`. 
In this case, I don't think you need a `repeat`.

```suggestion
  grid-template-columns: 1fr;
```